### ISSUE TITLE
Anchors used for YoWo Benchmark

### ISSUE THREAD

**Issue by Griffintaur** [MCG-NJU/MultiSports#16]
I am trying to reproduce the result using YOWO?
Can you let me know which anchors you used for YOWO benchmarks?

**Comment by MiaSanLei** [comment]
The anchors we used are [0.70458, 1.18803, 1.26654, 2.55121, 1.59382, 4.08321, 2.30548, 4.94180, 3.52332, 5.91979] as same as the anchors of [ucf24](https://github.com/wei-tim/YOWO/blob/master/cfg/ucf24.yaml) dataset in YOWO's codebase.

**Comment by MiaSanLei** [comment]
> I am trying to reproduce the result using YOWO? Can you let me know which anchors you used for YOWO benchmarks?

The anchors we used are [0.70458, 1.18803, 1.26654, 2.55121, 1.59382, 4.08321, 2.30548, 4.94180, 3.52332, 5.91979] as same as the anchors of ucf24 dataset in YOWO's codebase.

**Comment by MiaSanLei** [comment]
> I am trying to reproduce the result using YOWO? Can you let me know which anchors you used for YOWO benchmarks?

The anchors we used are [0.70458, 1.18803, 1.26654, 2.55121, 1.59382, 4.08321, 2.30548, 4.94180, 3.52332, 5.91979] as same as the anchors of [ucf24](https://github.com/wei-tim/YOWO/blob/master/cfg/ucf24.yaml) dataset in YOWO's codebase.

**Comment by Griffintaur** [comment]
Which pretrained model you used to initialize the 3D and 2D model in YoWo architecture?

**Comment by MiaSanLei** [comment]
> Which pretrained model you used to initialize the 3D and 2D model in YoWo architecture?
We use Darknet-19 [weights](http://pjreddie.com/media/files/yolo.weights) as our 2D pretrained model and resnext-101-kinetics.pth from [Google drive](https://drive.google.com/drive/folders/1zvl89AgFAApbH0At-gMuZSeQB_LpNP-M) as 3D pretrained.

**Comment by Griffintaur** [comment]
@MiaSanLei  sorry to ask another question, I was using the above settings as you described but `nan` was creeping in during training.
Did you change anything in ucf.yaml config file or anything to make it work for Multisports.
It would be wonderful if you could share the config file you used.

**Comment by MiaSanLei** [comment]
I modified the learning rate and classes in ucf24.cfg.







------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "MCG-NJU/MultiSports"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年2月16日(星期三) 上午10:24
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [MCG-NJU/MultiSports] Anchors used for YoWo Benchmark (Issue #16)





 
@MiaSanLei  sorry to ask another question, I was using the above settings as you described but nan was creeping in during training.
 Did you change anything in ucf.yaml config file or anything to make it work for Multisports.
 It would be wonderful if you could share the config file you used.
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
